src/components/FindWallet/WalletFilterSidebar/index.tsx

```diff
@@ -0,0 +1,265 @@
+      boxSize="1.875rem"
+    />
+  </Box>
+)
+
+export interface WalletFilterSidebarProps {
+  resetWalletFilter: MutableRefObject<() => void>
+  filters: Record<string, boolean>
+  updateFilterOption: (key: any) => void
+  updateFilterOptions: (key: any, value: any) => void
+}
+
+const WalletFilterSidebar: React.FC<WalletFilterSidebarProps> = ({
+  updateFilterOption,
+  ...restProps
+}) => {
+  const { filterOptions, setShowOptions } = useWalletFilterSidebar(restProps)
+
+  const filterPanelBg = useColorModeValue("primary100", "black400")
+  return (
+    <Accordion
+      as={VStack}
+      allowMultiple
+      reduceMotion
+      spacing={4}
+      alignItems="normal"
+      p={{ base: 4, sm: 0 }}
+      // Workaround to not having a dedicated prop to all items open by default
+      defaultIndex={Object.keys(filterOptions).map((key) => +key)}
+    >
+      {filterOptions.map((filterOption, idx) => {
+        return (
+          <AccordionItem
+            key={uniqueId("walletFilterSidebarItem")}
+            background={filterPanelBg}
+            borderRadius="base"
+            // Remove border color from global style
+            borderColor="transparent"
+            p={6}
+          >
+            {({ isExpanded }) => (
+              <>
+                <chakra.h3
+                  color="primary"
+                  borderBottom={isExpanded ? "1px" : "none"}
+                  borderColor="currentColor"
+                  m={0}
+                  pb={isExpanded ? 3 : 0}
+                  px={4}
+                >
+                  <AccordionButton
+                    color="inherit"
+                    fontWeight="inherit"
+                    fontSize="inherit"
+                    p={0}
+                    textAlign="initial"
+                  >
+                    <Box as="span" flex={1}>
+                      {filterOption.title}
+                    </Box>
+                    <AccordionIcon color="primary" boxSize={9} />
+                  </AccordionButton>
+                </chakra.h3>
+                <AccordionPanel as={List} p={0} m={0}>
+                  {filterOption.items.map((item, itemIdx) => {
+                    return (
+                      <Box
+                        borderBottom="1px"
+                        borderColor="lightBorder"
+                        pt="1.16rem"
+                        px={3}
+                        pb={3}
+                        _last={{ border: "none" }}
+                      >
+                        <SimpleGrid
+                          key={uniqueId("walletFilterSidebarItemPanel")}
+                          templateColumns="28px auto 34px"
+                          alignItems="center"
+                          columnGap={2.5}
+                          cursor="pointer"
+                          onClick={
+                            item.filterKey
+                              ? () => {
+                                  console.log(item.filterKey)
```

Comment:
Not sure if these were before but lets remove the consoles

Reply:
Whoops! That was me.